script/build.js

```diff
@@ -205,6 +214,6 @@ if (options.watch) {
 }
 
 smith.build(function(err) {
-    if (err) throw err;
-    console.log('Build finished!');
-  });
+  if (err) throw err;
+  console.log('Metalsmith build finished!  Now starting webpack...');
```

Comment:
Is this actually the right ordering? I thought webpack would have finished before this is executed...